GetDataType.js

```diff
@@ -1,11 +1,31 @@
-const dataTypes = [];
 function loadData(data) {
+  const dataTypes = [];
   const deff = data.library.statements.def;
   const filtered = deff.filter((d) => {
     return d.expression.operand !== undefined;
   });
+  function recursion(array) {
```

Comment:
A bit of a nitpick, but can we rename this function to something more specific? Maybe `traverseELM` or something idk